modules/azdo_ws2019agent/main.tf

```diff
@@ -1,8 +1,8 @@
 data "azurerm_shared_image_version" "existing" {
-  name                = "0.153.3"
-  gallery_name        = "UKHOSharedImageGallery"
+  name                = "0.154.3"
+  gallery_name        = "UkhoSharedImageGallery"
```

Comment:
Can UKHO be capitalised?

Reply:
I dont think it matters, the resource I created was UkhoSharedImageGallery